opentelemetry-api/opentelemetry/internal/version.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__version__ = "0.1.0"
```

Comment:
Why is this in `internal/` instead of the top level `opentelemetry/`?

Reply:
Put it in `internal/` since `version` is not part the specification yet.

We don't have a good way to put it under top level `opentelemetry/` if we want multiple packages to share the same namespace (namespace package).

Reply:
Also, as we make a namespace package, one could, in theory, have different versions of SDK and API subpackages, so it doesn't even make sense semantically to put the version out at the top level.